### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 504 (Gateway Timeout)

### DISCUSSION THREAD

**Discussion by ghost** [NG-ZORRO/ng-zorro-antd#7007] [Q&A]
I have a problem with your website.
![image](https://user-images.githubusercontent.com/91606641/137456250-0ef560cd-d91c-4ca9-978e-69f4e129907a.png)

**Comment by vthinkxie** [comment]
it should work now, thanks for your feedback

**Reply by izonnn** [reply]
Still not working... 
<img width="1461" alt="image" src="https://user-images.githubusercontent.com/17711140/137602037-1c8cfcf5-d583-4d89-8d1a-80c61b7cf0ae.png">
